packages/amplify_analytics_pinpoint/android/src/test/kotlin/com/amazonaws/amplify/amplify_analytics_pinpoint/AmplifyAnalyticsPinpointPluginTest.kt

```diff
@@ -66,28 +63,28 @@ class AmplifyAnalyticsPinpointPluginTest {
     @Test
     fun recordEvent_withProperties_returnsSuccess() {
 
-        val propertiesMap = hashMapOf<String, Any>(
-                "AnalyticsStringProperty" to "Pancakes",
-                "AnalyticsBooleanProperty" to true,
-                "AnalyticsDoubleProperty" to 3.14,
-                "AnalyticsIntegerProperty" to 42
+        val propertiesMap = hashMapOf(
+            "AnalyticsStringProperty" to "Pancakes",
+            "AnalyticsBooleanProperty" to true,
+            "AnalyticsDoubleProperty" to 3.14,
+            "AnalyticsIntegerProperty" to 42
         )
         val propertiesTypesMap = hashMapOf<String, Any>(
```

Comment:
Minor comment, but perhaps this should have been hashMapOf<String, String> instead of Any

Reply:
Fixed